src/stubber/minify.py

```diff
@@ -197,18 +225,6 @@ def minify_script(source_script: Union[Path, str, IO[str]], keep_report:bool=Tru
             ("rprint", 'self._log.info("Created stubs for'),
         ]
 
-    if not python_minifier:  # pragma: no cover
-        raise ModuleNotFoundError("python_minifier not available")
-
-    source_content = source_script
```

Comment:
I see that you have removed the ability to pass in a raw string input.

When I did #293, I added this check just to cover all the possible bases.
That said, accepting `Path` / string that is a path / raw string / `IO` is a bit redundant I suppose haha.

Reply:
Minify : 
 - inputs and Outputs allow Path, string and StringIO
 

cross-compile : 
 - inputs allow Path, string and StringIO
 - Outputs allow Path, and BytesIO